frontend/src/components/Modal.tsx

```diff
@@ -1,8 +1,37 @@
+import { useState } from "react";
 import styled from "styled-components";
 
-export const Modal = styled.div`
+const ModalBackground = styled.div`
   position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  justify-content: center;
+  align-items: center;
+  display: flex;
   background: rgb(0, 0, 0, 0.7);
-  width: 100%;
-  height: 100%;
+  backdrop-filter: blur(2px);
 `;
+
+const ModalBox = styled.div`
+  background: whitesmoke;
+  padding: 1em;
+  border-radius: 1em;
+`
+
+export type ModalProps = { children: string };
+export const Modal = ({children,}: ModalProps) => {
+  let [showModal, setShowModal] = useState(true);

```

Comment:
Use const here, it helps prevent accidental mutations to react controlled stated objects.

Reply:
professional let -> const linter

good idea